lib/application.js

```diff
@@ -232,23 +232,24 @@ const mixin = (application) => {
       const globals = (
         application.config.sandbox.global || DEFAULT_SANDBOX
       );
-      let i, len, moduleLink, moduleName, msg;
-      for (i = 0, len = globals.length; i < len; i++) {
-        moduleName = globals[i];
+      const len = globals.length;
+      for (let i = 0; i < len; i++) {
+        const moduleName = globals[i];
+        let moduleLink;
         if (moduleName === 'require') moduleLink = globalRequire;
         else moduleLink = global[moduleName];
         if (moduleLink) {
           if (moduleName in api.registry.deprecated) {
-            msg = api.registry.deprecated[moduleName];
+            const msg = api.registry.deprecated[moduleName];
             moduleLink = api.registry.deprecate(moduleLink, msg);
           }
           application.sandbox[moduleName] = moduleLink;
         }
       }
       const apis = application.config.sandbox.api || api.registry.defaultNames;
-      let j, jlen;
-      for (j = 0, jlen = apis.length; j < jlen; j++) {
-        moduleName = apis[j];
+      const jlen = apis.length;
+      for (let j = 0; j < jlen; j++) {
+        let moduleLink, moduleName = apis[j];
```

Comment:
`moduleLink` declaration should be moved to the next line, where it is also being initialized.